src/useForm.test.tsx

```diff
@@ -2832,4 +2832,55 @@ describe('useForm', () => {
       expect(resolverData).toEqual({ test: 'value' });
     });
   });
+
+  describe('mode with onTouched', () => {
+    it('should validate form only when input is been touched', async () => {
+      const Component = () => {
+        const { register, errors } = useForm({
+          mode: 'onTouched',
+        });
+
+        return (
+          <>
+            <input
+              type="text"
+              name="test"
+              ref={register({ required: 'This is required.' })}
+            />
+            {errors.test?.message}
+          </>
+        );
+      };
+
+      render(<Component />);
+
+      screen.getByRole('textbox').focus();
+
+      await actComponent(async () => {
+        await fireEvent.blur(screen.getByRole('textbox'));
+      });
+
+      expect(screen.getByText('This is required.')).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.queryByText('This is required.')).toBeInTheDocument();
```

I think you should `screen.queryByText()` because `getBy*` method throw error when specified element could not found.

Reply:
awesome! thanks @keiya01 🙏